src/inc/corjit.h

```diff
@@ -292,7 +292,7 @@ class ICorJitCompiler
 class ICorJitInfo : public ICorDynamicInfo
 {
 public:
-    // return memory manager that the JIT can use to allocate a regular memory
+    // OBSOLETE: return memory manager that the JIT can use to allocate a regular memory
```

Comment:
Are you still testing the CoreCLR JIT against .NET Framework, or can we start deleting unused JIT/EE interface methods?

Reply:
We still do, we do not have ILGen and auto spmi collections/replays on GitHub.